### ISSUE TITLE
ability to enable/disable/toggle key map group

### ISSUE THREAD

**Issue by sds100** [keymapperorg/KeyMapper#397]

**Comment by sds100** [comment]
Don't think we need this and would bloat the UI with even more buttons. A user can just select all key maps in the group. One user did email they wanted this but found it sufficient do select all etc.

**Comment by minh2134** [comment]
Right, to reiterate my usecase in #1723: I think it would be pretty neat to be able to toggle subgroup (with respect also to constraint), but also able to bind a keymap to it. This would be pretty helpful to switch keymaps inside a constraint (say, within an app) at will.

**Comment by sds100** [comment]
I will implement this. There is space for a switch next to the group constraints AND/OR buttons

**Comment by sds100** [comment]
implemented

**Comment by cinghia1992** [comment]
> implemented

Sorry for replying to a request from 2025, but has it been implemented? I don't see anything in the app that allows me to activate or deactivate a subgroup using a specific key combination as a trigger.

**Comment by jambl3r** [comment]
@cinghia1992 Hello, yes this was implemented in 4.0.0+. This version is still being rolled out so your device should give you the option to update within the next few weeks. If you don't want to wait, you can sign up to the Google Play beta programme or download the FOSS version directly from GitHub.

**Comment by jakenvac** [comment]
Hi, Is there a way to toggle these groups with another keybind? Perhaps via an intent? I can't see anything documented on the wiki.

I think this would be a great use case for togglable "layers" (think caps lock or symbol layers)

**Comment by jambl3r** [comment]
Not currently, I created a new issue for this #2075
